assignments/this.js

```diff
@@ -1,26 +1,62 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Window Binding
+* 2. Implicit Binding
+* 3. New Binding
+* 4. Explicit Binding
 *
 * write out a code example of each explanation above
 */
 
 // Principle 1
-
 // code example for Window Binding
+let me = {
+    name: "Tristan", 
+    age: 27,
+    sayName: function() {
+        console.log(this.name);
+    }
+};
+
+sayName();
 
-// Principle 2
 
+// Principle 2
 // code example for Implicit Binding
 
-// Principle 3
+let me = {
+    name: "Tristan", 
+    age: 27,
+    sayName: function() {
+        console.log(this.name);
+    }
+};
+
+me.sayName();
 
+// Principle 3
 // code example for New Binding
 
+let person = function(sex, name, age) {
+    this.sex = sex;
+    this.name = name;
+    this.age = age;
+};
+
+const tristan = new person("Male", "Tristan", 27);
```

Comment:
Make sure when you're building a "pseudo-classical" object in this way you capitalize Person

Reply:
It's just convention.